### PR TITLE
Pin amazon.aws collection in before_install

### PR DESCRIPTION
**Pull request by mdellweg**
[noissue]

### CODE REVIEW

File: templates/github/.github/workflows/scripts/before_install.sh.j2 [hunk 1]
```diff
@@ -158,7 +158,7 @@ pip install docker netaddr boto3 ansible
 
 for i in {1..3}
 do
-  ansible-galaxy collection install amazon.aws && s=0 && break || s=$? && sleep 3
+  ansible-galaxy collection install "amazon.aws:1.5.0" && s=0 && break || s=$? && sleep 3
```
Comment (fao89) [inline templates/github/.github/workflows/scripts/before_install.sh.j2:161]: it seems they reverted your fix!

**Review by fao89 — APPROVED** [review]
thank you!

**Review by fao89 — APPROVED** [review]

File: templates/github/.github/workflows/scripts/before_install.sh.j2 (hunk 1 above)
Comment (mdellweg) [reply templates/github/.github/workflows/scripts/before_install.sh.j2:161]: Damn it we are back where we started?

Comment (fao89) [reply templates/github/.github/workflows/scripts/before_install.sh.j2:161]: not sure, but I believe so

**Comment by fao89** [comment]
FYI @rochacbruno

File: templates/github/.github/workflows/scripts/before_install.sh.j2 (hunk 1 above)
Comment (mdellweg) [reply templates/github/.github/workflows/scripts/before_install.sh.j2:161]: I don't see it reverted. This may actually be a new bug.

**Pull request merged**